core/API/DataTableManipulator/ReportTotalsCalculator.php

```diff
@@ -66,14 +66,19 @@ protected function manipulateDataTable($dataTable)
         $firstLevelTable    = $this->makeSureToWorkOnFirstLevelDataTable($dataTable);
         $metricsToCalculate = Metrics::getMetricIdsToProcessReportTotal();
 
+        $realMetricNames = array();
```

Comment:
This is all done to make it faster, eg do not get readable column name as often. This here is executed quite often